datumaro/plugins/splitter.py

```diff
@@ -13,34 +14,172 @@
 
 NEAR_ZERO = 1e-7
 
+SplitTask = Enum(
+    "split", ["classification", "detection", "segmentation", "reidentification"]
```

Comment:
Consider adding an alias `reid` or renaming to this name.

Reply:
I modified that part. #1d45ec6